### PR TITLE
lesson 1 complete

### PR DESCRIPTION
**Pull request by Pisotski**

### CODE REVIEW

File: src/index.css [hunk 1]
```diff
@@ -1,68 +1,43 @@
...
+	border: 0;
+	font-size: 100%;
+	font: inherit;
+	vertical-align: baseline;
+}
+/* HTML5 display-role reset for older browsers */
+article, aside, details, figcaption, figure, 
+footer, header, hgroup, menu, nav, section {
+	display: block;
 }
```
Comment (ethan-carlson) [inline src/index.css:25]: This is very strange css.  What is the purpose?

File: src/App.tsx [hunk 2]
```diff
@@ -1,35 +1,41 @@
-import { useState } from 'react'
-import reactLogo from './assets/react.svg'
-import viteLogo from '/vite.svg'
+import * as React from 'react';
 import './App.css'
 
-function App() {
-  const [count, setCount] = useState(0)
+// Go ahead and render the item’s url, author, num_comments, and points as well. 
+// In the special case of the url, use an HTML anchor HTML element (read: <a> tag) that surrounds the title.
```
Comment (ethan-carlson) [inline src/App.tsx]: Would remove comments that don't apply to the code.

**Review by ethan-carlson — APPROVED** [review]
Looks good to me!

Unless you tell me otherwise, I will review your code as I would a professional, which means I will probably be a little stricter than is expected of you in this course.  Please take my comments just as directions to further improve code quality and readability, not as an indication that you are not meeting the expectations of the class.  If you don't meet the assignment rubric, I will request changes on the PR.

My general feedback on this PR is to avoid "cruft" - code that isn't doing anything.  It's very difficult to remember to go back and delete things later, so it's best practice to never merge anything that doesn't have a purpose.

File: src/App.tsx (hunk 2 above)
Comment (vlad-pisotskyi) [reply src/App.tsx]: deleted

File: src/index.css (hunk 1 above)
Comment (vlad-pisotskyi) [reply src/index.css:25]: An old CSS resetter before I start using styling modules. I can remove it if you need it.

**Pull request merged**